pkg/machine/qemu/machine_test.go

```diff
@@ -4,8 +4,15 @@
 package qemu
 
 import (
+	"encoding/base64"
+	"fmt"
+	"strings"
 	"testing"
 
+	"github.com/containers/podman/v4/pkg/machine"
+
```

Comment:
These empty lines should not be here.

Reply:
I will remove them. Is there a way to avoid running all the checks again?

Reply:
No